gtest/src/log.rs

```diff
@@ -396,6 +396,57 @@ impl RunResult {
             .flat_map(DecodedCoreLog::try_from_log)
             .collect()
     }
+
+    /// If the main message panicked.
+    pub fn panicked(&self) -> bool {
+        if self.log.is_empty() {
+            return false;
+        }
+        matches!(
+            self.log
+                .last()
+                .expect("Checked for empty log previously")
```

Comment:
you don't have any ordering guarantees.

Reply:
search by reply_to(main_message)

Reply:
log.id == result.message_id?

Reply:
> log.id == result.message_id?

reply_to(), not log.id

Reply:
I cannot find that function in Log or CoreLog?

Reply:
> I cannot find that function in Log or CoreLog?

As I see we cut reply details into just code while converting core type into gtest Log, we mustn't do it: I guess this PR should solve it. 
As well as we can't just take any reply_code.is_some() message, because it may be other one.

Reply:
Oh so should I find where this is a thing in the actual execution and add it onto CoreLog? The reply_to: MessageId

Reply:
Ah yes I found it its super simple I see